cmd/snap-gpio-helper/cmd_unexport.go

```diff
@@ -31,5 +29,18 @@ type cmdUnexportChardev struct {
 }
 
 func (c *cmdUnexportChardev) Execute(args []string) error {
-	return errors.New("not implemented")
+	aggregatedChip, err := removeGadgetSlotDevice(c.Args.Gadget, c.Args.Slot)
```

Comment:
Since we are running other programs I think this could use `signal.NotifyContext` and pass that context to all the functions that need it.

Reply:
Unexport path doesn't have places that could make use of a context unlike the export path.